mrq/version.py

```diff
@@ -1,4 +1,4 @@
-VERSION = "0.9.10"
+VERSION = "0.9.29"
```

Comment:
Next version is 0.9.30

Reply:
I haven't yet changed the version in this PR. That and over half of the items in the changes area are from commits that were applied upstream and now are getting re-applied on our repo. I'll post the actual changes.